src/sync/installation.ts

```diff
@@ -337,7 +330,6 @@ export const handleBackfillError = async (
 	err,
 	data: BackfillMessagePayload,
 	nextTask: Task,
-	subscription: Subscription,
```

Comment:
I don't understand the purpose of removing `subscription` and find it again inside the `markCurrentTaskAsFailedAndContinue`. Is it for future to extra the error handler to outside?